### DISCUSSION TITLE
how to set cut-off

### DISCUSSION THREAD

**Discussion by zjd11** [deepmodeling/deepmd-kit#1346] [Q&A]
Hi, I,m training a model for a cell with length of 10*10*10 Ang, whether the cut-off (rcut) should be smaller than 5 Ang, as molecular dynamics requires，Because the obtained model need to be used in Deep  molecular dynamics to explore new structure of the cell to improve the model. thank you

**Comment by wanghan-iapcm** [comment]
DeePMD-kit does not require that the rc is smaller than half cell size.
